IbisVTK/vtkQt/CMakeLists.txt

```diff
@@ -35,32 +34,3 @@ if( IBIS_INSTALL_LIBS_AND_HEADERS )
     INSTALL_TARGETS(/lib vtkQt)
     INSTALL_FILES(/include FILES ${IBIS_VTK_QT_HDR} )
 endif( IBIS_INSTALL_LIBS_AND_HEADERS )
-
```

Comment:
Why are we getting rid of Plugins for QtCreator ?

Reply:
Do you mean QT_EXTENSIONS_INCLUDE_DIR?
It is not defined or used  anywhere.